ten-simple-rules-dockerfiles.Rmd

```diff
@@ -617,14 +656,14 @@ Together we can develop common practices and shared materials for better transpa
 
 # Acknowledgements {#acknowledgements .unnumbered}
 
-DN is supported by the project Opening Reproducible Research II (\href{https://o2r.info/}{https://o2r.info/}; \href{https://www.uni-muenster.de/forschungaz/project/12343}{https://www.uni-muenster.de/forschungaz/project/12343}) funded by the German Research Foundation (DFG) under project number PE 1632/17-1.
+DN is supported by the project Opening Reproducible Research II (\href{https://o2r.info/}{https://o2r.info/}; \href{https://www.uni-muenster.de/forschungaz/project/12343}{https://www.uni-muenster.de/forschungaz/project/12343}) funded by the German Research Foundation (DFG) under project number PE 1632/17-1.  DN and SJE are supported by a Mozilla mini science grant.
 
 # Contributions {#contributions .unnumbered}
```

Comment:
@vsoch is running on fumes because she is a research software engineer with no way to self-fund other than begging for... oh wait nevermind. :)